src/messages/HeadersMessage.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * This file is part of the bitcoin-classic project
+ * Copyright (c) 2009-2010 Satoshi Nakamoto
+ * Copyright (c) 2009-2015 The Bitcoin Core developers
+ * Copyright (C) 2017 Tom Zander <tomz@freedommail.ch>
+ * Copyright (C) 2017 Angel Leon <@gubatron>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef BITCOINCLASSIC_HEADERSMESSAGE_H
+#define BITCOINCLASSIC_HEADERSMESSAGE_H
+
+#include "NetworkMessage.h"
+
+namespace {
+    class HeadersMessage : public NetworkMessage {
+    public:
+        ~HeadersMessage() {}
```

Comment:
no need to write an empty destructor, the compiler will write that for you if you omit it. The only reason to write such a thing is if you want to mark it virtual, which is only relevant in a baseclass.